store/populations.go

```diff
@@ -592,6 +594,7 @@ func (s *store) GetStats(ctx context.Context, in *pb.GetStatsRequest,
 			keySuffix += "^" + p + "^" + v
 		})
 	}
+	log.Println(keySuffix)
```

Comment:
Intend to keep it?

Reply:
Yes, keep it for easier debugging before we fix the issue.